src/network/core/tcp.cpp

```diff
@@ -236,7 +236,13 @@ bool NetworkTCPSocketHandler::CanSendReceive()
 	FD_SET(this->sock, &write_fd);
 
 	tv.tv_sec = tv.tv_usec = 0; // don't block at all.
-	if (select(FD_SETSIZE, &read_fd, &write_fd, nullptr, &tv) < 0) return false;
+
+#ifndef __EMSCRIPTEN__
+	int nfds = FD_SETSIZE;
```

Comment:
again, if this is because EMSCRIPTEN doesn't define `FD_SETSIZE`, it should be set separately, instead of creating an extra variable

Reply:
It does define it to 1024, but its `select` support only 64
https://github.com/emscripten-core/emscripten/issues/1711

Reply:
hrm, well just a significant comment explaining why then, probably with a link to the bug